docs/Installation/System-Requirements.md

```diff
@@ -0,0 +1,29 @@
+title: System Requirements
+description: System requirements to sync and run Pantheon 
+<!--- END of page meta data -->
+
+# System Requirements 
+
+The system requirements for Pantheon depend on a number of factors: 
+
+* Network - are you connecting to a public Ethereum network (eg, mainnet) or a private network 
+* Network size if connecting to a private network 
+* Number and complexity of queries being performed by node 
```

Comment:
`performed by node` doesn't make a lot of sense to me. 
```suggestion
* Number and complexity of JSON-RPC, PubSub or GraphQL queries being handled by the node
```

Reply:
Fixed